ext/ddtrace.c

```diff
@@ -2136,6 +2136,55 @@ static void dd_ensure_root_span(void) {
     }
 }
 
+/* {{{ proto void dd_handle_fork(): void */
+PHP_FUNCTION(dd_handle_fork) {
+    UNUSED(execute_data);
+    UNUSED(return_value);
+    // CHILD PROCESS
```

Comment:
I assume it's still todo to deduplicate with the code in handlers_pcntl.c?

Reply:
What do you exactly mean by "deduplicate"? Do you mean some refactoring duplicated code (using this function in the pcntl code)?

Reply:
Yes, refactoring the common code out into a single function.